Gaijinizer patch/Map050.txt

```diff
@@ -505,8 +505,8 @@ just our direction.#MTLed
 > CONTEXT: Map050/events/46/pages/1/228/Dialogue/yuuma
 > CONTEXT: Map188/events/46/pages/0/295/Dialogue/yuuma
 > CONTEXT: Map188/events/46/pages/1/228/Dialogue/yuuma
-What does \\N[1] do? It will be somewhat
-roundabout, but will you go another way?#MTLed
+What will you do, \\N[1]? It'll be quite a detour
```

Comment:
Let's try this:
>**_What will you do, \\\N[1]?
It'd be quite a detour, but are you going
to try a different way?_**